src/foam/support/view/SupportEmailView.js

```diff
@@ -119,24 +123,28 @@ foam.CLASS({
   methods: [
     function initE(){
       var self = this;
-      this.supportEmailDAO.limit(1).select().then(function(a){ 
+      this.user.supportEmails.limit(1).select().then(function(a){ 
         self.emptyDAO = a.array.length == 0;
       });
 
       this
       .addClass(this.myClass())
       .start().addClass('Rectangle-11-Copy')
         .start().addClass('title1')
-          .start().add('Support Email Management').addClass('title').end()
+          .start()
+            .add('Support Emails Management').addClass('title')
```

Comment:
Let's make a message for this.

Reply:
`.start().add('Support Email Management').addClass('title').end()` is equal to `<div class='title'>Support Email Management</div>`. Is it necessary to put this string in a message variable when we only use this string once?

If we put `Support Email Management` in a message, we may need to put `No support email connected` in the message, as well, for consistency.

Reply:
Will update this.